docs/development-guidelines/global-deployment.md

```diff
@@ -1,18 +1,32 @@
 # Global Deployment
 
-This document outlines a workaround for a known upstream bug handling global-standard type Azure OpenAI deployments (issue of resource deletion conflicts).
+This document outlines the migration guidance for transitioning to a global deployment SKU for GPT models in Azure OpenAI, as well as troubleshooting steps for handling resource deletion conflicts.
 
-## Issue
+## Migration Guidance
```

Comment:
It will be great to add a link to a sample repo or PR

Reply:
I agree. I prefer code snippets since repos could be archived or it may be more confusing in that context.

Reply:
Added code snippets.